### DISCUSSION TITLE
Applying forecastDataPoints to a single series

### DISCUSSION THREAD

**Discussion by pursuitofloot** [apexcharts/apexcharts.js#2775] [Q&A]
Hi there, is it possible to apply [forecastDataPoints](https://apexcharts.com/docs/options/forecastdatapoints/) to a single series?

I have a line chart with 4 series of data, but only 1 of them I want to style with the dotted lines to show it's forecast data. As this is not specifically mentioned in the docs as a feature, I figured I'd ask here in case it was achievable in some way.

Thanks in advance 🙏

**Comment by hellqvistjohan** [comment]
Hi, i have this need as well, i would like to be able to exclude one or more series from forecastDataPoint.
